src/SciMLBase.jl

```diff
@@ -4,6 +4,7 @@ using ConstructionBase
 using RecipesBase, RecursiveArrayTools, Tables
 using DocStringExtensions
 using LinearAlgebra
+using Interpolations
```

Comment:
This shouldn't be in SciMLBase if we can avoid it.

Reply:
What is the load time impact?

Reply:
With:   
```
  5.509606 seconds (4.51 M allocations: 333.874 MiB, 0.22% gc time, 2.26% compilation time)

  0.000070 seconds (104 allocations: 8.281 KiB)

  0.000054 seconds (104 allocations: 8.281 KiB)

  0.000109 seconds (104 allocations: 8.281 KiB)

  0.000069 seconds (104 allocations: 8.281 KiB)
```

Without:
```
  3.993145 seconds (1.89 M allocations: 146.430 MiB, 0.34% gc time, 3.52% compilation time)

  0.000104 seconds (104 allocations: 8.281 KiB)

  0.000058 seconds (104 allocations: 8.281 KiB)

  0.000069 seconds (104 allocations: 8.281 KiB)

  0.000104 seconds (104 allocations: 8.281 KiB)
```

NB. Couldn't use btime since `using` has to be at top level, is there a better way to do this?